sources/install.sh

```diff
@@ -2340,6 +2340,20 @@ function install_ligolo-ng() {
   add-test-command "ligolo-proxy --help"
 }
 
+function install_nfspy() {
+  colorecho "Installing nfspy"
+  git -C /opt/tools/ clone https://github.com/bonsaiviking/NfSpy.git
+  cd /opt/tools/NfSpy
+  apt install -y libfuse-dev
+  python2 -m pip install fuse-python
```

Comment:
In order to work in an isolated environment, can you add a python venv like this ?

```bash
virtualenv -p /usr/bin/python2 ./venv
./venv/bin/python2 -m pip install fuse-python
./venv/bin/python2 setup.py install
chmod +x scripts/*
sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspy
sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspysh
ln -v -s /opt/tools/NfSpy/scripts/nfspy /opt/tools/bin/nfspy
ln -v -s /opt/tools/NfSpy/scripts/nfspysh /opt/tools/bin/nfspysh
```